Standards/scs-0102-v1-image-metadata.md

```diff
@@ -92,7 +92,7 @@ as referenced by its name.
 Note the _at least_ wording: Providers can replace images more often.
 The frequency is starting from the first release; so an image published on 2021-04-14 with an
 update frequency of `monthly`, should be replaced no later than 2021-05-14. Due to weekends
-etc., up to 3 days later is not considered a violation of this policy. So the a valid sequence
+etc., up to 3 days later is not considered a violation of this policy. So a valid sequence
```

Comment:
... not considered _as_ a violation of this policy ...?

Reply:
done

Reply:
Sorry, but IMO that was correct initially. https://duckduckgo.com/?t=ffab&q=%22considered+a+violation%22&ia=web

Reply:
Oh ok. Thanks for checking.

Reply:
change reverted.